aspnetcore/blazor/components/render-components-outside-of-aspnetcore.md

```diff
@@ -35,19 +35,18 @@ In the console app's project file (`ConsoleApp1.csproj`), update the console app
 + <Project Sdk="Microsoft.NET.Sdk.Razor">
 ```
 
-In a command shell, add a Razor component to the project:
+Add the following `RenderComponentExample` component to the project.
 
-```dotnetcli
-dotnet new razorcomponent -n Component1
-```
+`RenderComponentExample.razor`:
 
 ```razor
-<h1>Component1</h1>
+<h1>Render Component Example</h1>
 
-<p>Hello from Component1!</p>
+<p>@Message</p>
 
 @code {
-
+    [Parameter]
+    public required string Message { get; set; }
```

Comment:
```suggestion
    public string Message { get; set; }
```
The `required` keyword should never be used with Blazor component parameters. It doesn't actually make the parameter "required", even though nullability analysis seems to indicate so.

We even have an issue being tracked to add an analyzer warning when doing this: https://github.com/dotnet/aspnetcore/issues/47864.

Maybe it would be worth calling this out somewhere in the docs, too?

Reply:
This will be tracked by a new issue at https://github.com/dotnet/AspNetCore.Docs/issues/29236.